source/tasks/CreateOctopusRelease/CreateOctopusReleaseV3/task.json

```diff
@@ -172,7 +180,10 @@
         {
             "target": "ProjectName",
             "endpointId": "$(OctoConnectedServiceName)",
-            "dataSourceName": "OctopusAllProjects",
+            "dataSourceName": "OctopusListProjects",
```

Comment:
We pull the Projects dropdown data from our list endpoints instead, filtering by our filter (which may be an empty string)